dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/ApplicationManager.java

```diff
@@ -113,6 +114,9 @@ protected void modified(ComponentContext compcontext, Map<String, Object> proper
         setStopTimeout(stopTimeoutValue);
         ApplicationStateCoordinator.setApplicationStartTimeout(startTimeoutValue);
         ApplicationStateCoordinator.setApplicationStopTimeout(stopTimeoutValue);
+
+        String targetExpandDir = (String) properties.get("targetDir");
+        setTargetDir(targetExpandDir);
```

Comment:
The target directory needs to be set in 'modified()' too.. Given that this method is already calling modified it's probably best to move most of the activate method into the modified method